.github/linters/urunc-dict.txt

```diff
@@ -342,3 +342,4 @@ yourdockerhubid
 zerolog
 overlayfs
 macaddr
+qboot
```

Comment:
Is this change required? I do not see any reference to qboot in the rest of the changes.